shelter/README.md

```diff
@@ -75,7 +75,7 @@ OPTIONS:
    --verifier       set the type of quote verifier(nullquote, sgx_la or sgx_ecdsa)
    --tls            set the type of tls wrapper(nulltls, wolfssl or wolfssl_sgx)
    --crypto         set the type of crypto wrapper(nullcrypto, wolfcrypt or wolfcrypt_sgx)
-   --mutual         set to enable mutual attestation(True, False)
+   --mutual         enable mutual attestation by '--mutual' or disable it by ignoring this option
```

Comment:
Just simply "enable mutual attestation" is enough.

Reply:
done